client-js/app/scripts/rpc.ts

```diff
@@ -92,6 +92,7 @@ export class JsonRpcSocket {
    * Close the rpc socket by closing the underlying WebSocket.
    */
   close(code?: number, reason?: string) {
+    code = code || 1000;
```

Comment:
This is basically saying: if no code is provided, set code to 1000. Are there other codes? Are you just planning for the future?